wg-component-standard/README.md

```diff
@@ -33,4 +33,17 @@ Develop a standard foundation (philosophy and libraries) for core Kubernetes com
     - [@kubernetes/wg-component-standard](https://github.com/orgs/kubernetes/teams/wg-component-standard) - Component Standard Discussion
 <!-- BEGIN CUSTOM CONTENT -->
 
+## Issues
+* [Project Board](https://github.com/orgs/kubernetes/projects/26)
+* [k/k Issues / PR's needing Review](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=label%3Awg%2Fcomponent-standard+)
```

Comment:
just a suggestion: could we rephrase `k/k` to `kubernetes/kubernetes` or something like `PRs needing review in the Kubernetes repo`? Newcomers might not be familiar with the k/k jargon. :grimacing:

Reply:
@nikhita Thanks for the feedback -- I've removed the several abbreviated  references to `k/{repo}`.